### ISSUE TITLE
resize inverts x and y

### ISSUE THREAD

**Issue by ngreenwald** [vanvalenlab/deepcell-toolbox#31]
The current resize function swaps the rows and columns when resizing single-channel data. The test_resize function will fail if you modify the input data to be non-square. 

It looks like @msschwartz21 got around this in applications by [swapping the x/y ](https://github.com/vanvalenlab/deepcell-tf/blob/41dd06653ae892471398cccc79299f6ef4bbdf56/deepcell/applications/application.py#L244)coordinates when values are passed in. However, I think it would be better to take care of this within resize, as this led to some very confusing bugs when I tried to use resize outside of the application.

**Comment by willgraf** [comment]
I recall we use either `skimage` or `cv2` to do the resizing. Do we know if they both invert the axes?

**Comment by ngreenwald** [comment]
I think only CV2 does the inverting, which is why this issue only pops up with single channel data, since we use skimage for multichannel

**Comment by willgraf** [comment]
Ah thanks, I missed that in the original post.

Could we just swap the axes before going into resize?

```python
resize = lambda d: np.expand_dims(cv2.resize(np.squeeze(d), shape[::-1],
                                                      interpolation=interpolation),
                                           axis=channel_axis)
```

**Comment by ngreenwald** [comment]
Nice, new tests seem to cover all the bases.

**Comment by willgraf** [comment]
Closed by #32
